pages/dkp/kommander/1.3/projects/project-deployments/index.md

```diff
@@ -0,0 +1,99 @@
+* Avoid configuration drift. The scope of manual changes made by operators expands over time. It soon becomes difficult to know which cluster configuration is critical and which is left over from temporary workarounds or live debugging. Over time, changing a project configuration or replicating a deployment to a new environment becomes a daunting task. GitOps supports simple, reproducible deployment to multiple different clusters by having a single source of truth for cluster and application configuration.
+
+That said, there are some cases when live debugging is necessary in order to resolve an incident in the minimum amount of time. In such cases, a pull request-based workflow adds expensive overhead when you need it least. Dispatch's CD strategy supports this scenario by letting you disable the Auto Sync feature. After Auto Sync is disabled, Dispatch will stop synchronizing the cluster state from the GitOps git repository. This lets you use kubectl, helm or whichever tool you need to resolve the issue.
+
+## Prerequisites
+
+This section requires that Dispatch is installed, and, optionally, you can configure Dispatch to be deployed in "CD-only" mode by modifying the `cluster.yaml` file as follows:
+
+```yaml
+- name: dispatch
+  enabled: true
+  values: |
+    ci:
+      enabled: false
+```
+
+## Continuous Deployments
+
+After enabling Dispatch and [configuring your project and its clusters](../), navigate to the a **Continuous Deployment (CD)** tab under your Project. Here, you can create a GitOps source which is an SCM repository hosting the application definition. It is recommended to create a secret first followed by a GitOps source that is backed by the secret:
+
+![Continuous Deployment (CD)](/dkp/kommander/1.3/img/project-cd-welcome-screen.png)
+
+Before we setup our GitOps source, we need to create a secret that can be used to access our GitOps source.
+
+### Setting up a secret for accessing GitOps source.
+
+Go ahead and create a secret to be used by Dispatch to read the contents of your GitOps repository by creating SCM webhooks:
+
+![Secret for GitOps source](/dkp/kommander/1.3/img/project-cd-secret-create.png)
+
+Refer to this guide to learn more about [setting up credentials](/dkp/dispatch/1.4/tutorials/ci_tutorials/credentials#setting-up-github-credentials) for Dispatch. Note that the above screen creates a `types.kubefed.io/v1beta1, Kind=FederatedSecret` and this is not yet supported by Dispatch CLI. Please use the GUI as shown above to create a federated secret OR create a `FederatedSecret` manifest and apply it to the project namespace. Learn more about [FederatedSecrets](../project-secrets/).
+
+### Creating GitOps Source
+
+Once the secret is created, you should be able to view it under the `Secrets` tab. Go ahead and configure the GitOps source backed by this secret:
```

Comment:
can we remove all the cases of **should** and replace with **will**? should sounds like we are hedging our bets if this will indeed work as described :-)

Reply:
We do try to avoid future tense - "will" - where possible.

We also want to avoid passive-voice since it obscures WHO is doing the WHAT.

Starting a sentence with "Once..." is better phrased as "After..." for a variety of reasons. "Go ahead and <verb>..." is the same as starting the sentence with just, "<Verb> something...."

So, suggested rewording to incorporate all of this: "After you create the secret, you can view it under the `Secrets` tab. Configure the the GitOps source backed by this secret:"